ui/pages/send/send-content/send-hex-data-row/send-hex-data-row.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import {
+  getSendHexData,
+  isHexDataError,
+  updateSendHexData,
+} from '../../../../ducks/send';
+import SendRowWrapper from '../send-row-wrapper';
+import { useI18nContext } from '../../../../hooks/useI18nContext';
+
+const SendHexDataRow = () => {
+  const t = useI18nContext();
+
+  const data = useSelector(getSendHexData);
```

Comment:
How about:
```suggestion
  const { input, error } = useSelector(getSendHexData);
```

instead of creating an additional  `isHexDataError` selector?